cloud-functions/cf-dsm-file-scanner/dsm-gcp-setup/dsm-setup.sh

```diff
@@ -65,4 +65,12 @@ echo "Setting the cors config"
 ./cors-config.sh $1
 
 echo "Setting permissions for the ddp-downloader Service Account"
-./update-download-service-account $project_name
+./update-download-service-account.sh $project_name
```

Comment:
Nit: Line 36 empty quotes.  Do we need them?

Reply:
Yes, that means to not create subscriptions at this point. Those subscriptions need to be setup after the service is created and service url is generated , so we can set them up as PUSH to those urls